alphadia/calibration/plot.py

```diff
@@ -8,15 +8,82 @@
 
 import numpy as np
 from matplotlib import pyplot as plt
-
-from alphadia.plotting.utils import density_scatter
+from scipy.stats import gaussian_kde
 
 if TYPE_CHECKING:
     import pandas as pd
 
     from alphadia.calibration.estimator import CalibrationEstimator
 
 
+def _density_scatter(
```

Comment:
I think this is the first time i've seen you making a module larger rather than smaller.
The overall goal is to resolve the mixed bag `alphadia.plotting.utils` ?